Snippets/Snippets_6/DataBus/CleanupStrategy/Usage.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Snippets5.DataBus.CleanupStrategy
```

Comment:
this looks wrong. Copy paste error? Shouldn't this be Snippets_6 and not Snippets5?